emulator.py

```diff
@@ -38,6 +38,54 @@ def index():
     return "OK"
 
 
+# Needs to be defined in emulator.py to keep context of flask and db global variables
+def retry_test(method):
+    db.insert_supported_methods([method])
+
+    def decorator(func):
+        @wraps(func)
+        def wrapper(*args, **kwargs):
+            response_handler = testbench.common.handle_retry_test_instruction(
+                db, flask.request, method
+            )
+            return response_handler(func(*args, **kwargs))
+
+        return wrapper
+
+    return decorator
+
+
+@root.route("/retry_tests", methods=["GET"])
+def list_retry_tests():
+    response = json.dumps({"retry_test": db.list_retry_tests()})
+    return flask.Response(response, status=200, content_type="application/json")
+
+
+@root.route("/retry_test", methods=["POST"])
+def create_retry_test():
+    payload = json.loads(flask.request.data)
+    test_instruction_set = payload.get("instructions", None)
+    if not test_instruction_set:
+        return flask.Response("instructions is not defined", status=400)
```

Comment:
Add `content_type="text/plain"` to this response?

Reply:
Done.